marklogic-langchain4j/src/main/java/com/marklogic/langchain4j/splitter/JsonChunkDocumentProducer.java

```diff
@@ -15,10 +15,11 @@
 import com.marklogic.langchain4j.embedding.Chunk;
 import com.marklogic.langchain4j.embedding.DocumentAndChunks;
 import com.marklogic.langchain4j.embedding.JsonChunk;
+import com.smartlogic.classificationserver.client.ClassificationException;
+import com.smartlogic.classificationserver.client.ClassificationScore;
 import dev.langchain4j.data.segment.TextSegment;
 
-import java.util.ArrayList;
-import java.util.List;
+import java.util.*;
 
 class JsonChunkDocumentProducer extends AbstractChunkDocumentProducer {
```

Comment:
Make the two ChunkDocumentProducers receive a TextClassifier as a constructor arg instead of being part of ChunkConfig - the latter is intended to be simple options provided by a user, whereas a TextClassifier should be a constructor dependency.

Reply:
Ignore this for now, we can refactor later.